spinalcordtoolbox/deepseg/models.py

```diff
@@ -1,6 +1,7 @@
 # coding: utf-8
 """
 Deals with models for deepseg module. Available models are listed under MODELS.
+Important: model names (onnx or pt files) should have the same name as the enclosing folder.
 """
 
 
```

Comment:
Just following up on this PR. I notice in the TODOs you mention:

> * Create new ivadomed release
> * Update `setup.py` in SCT to update ivadomed version to the latest

Ivadomed 2.9.2 has been recently released, and also, currently SCT is not pinning the version of ivadomed anymore (so updates are automatic):

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/042cb51a7bb555e1ace55e8d4c3e7b4e293056f3/setup.py#L36

So, on `master`, right now we have the current version of ivadomed already getting installed:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/4900337927?check_suite_focus=true#step:4:29

I think that means both these points have been addressed, and we just need to work out the `_seg` suffix, right?

Reply:
Hi @jcohenadad! I wanted to offer to help with the suffix issue (just to help move this PR along).

From what I understand, on the ivadomed side of things, we eventually settled on the suffix `_class-0_pred`, `_class-1_pred`, etc. (https://github.com/ivadomed/ivadomed/issues/1006#issuecomment-1009211678)

Is this a convention we want to adopt on SCT's end of things, too?

Reply:
@joshuacwnewton good call! but fortunately this won't affect us because this change only affects multi-class models. However in the future, if we have multi-class models, that could be problematic.

Actually, I think we _already_ have multiclass tasks: `seg_tumor-edema-cavity_t1-t2`, `seg_gm_sc_7t_t2star` (and maybe more). So we should probably check these 😬 . Also, maybe we should consider adding simple tests to each model (eg: checking if at least the function produces the expected output files).